reference/installation/README.md

```diff
@@ -0,0 +1,32 @@
+# Generating API reference docs
+
+The api.html doc in this directory is generated using https://github.com/caseydavenport/gen-crd-api-reference-docs.
```

Comment:
Thanks for documenting this! But I'm wondering whether if location is the place for this information. It's useful for us but maybe not docs users. WDYT about `CONTRIBUTING_DOCS.md`?

Reply:
This doc does not show up in the website so I don't users would come across it. I'd like to keep it here so it is obvious how to generate this doc.